src/index.html

```diff
@@ -1,13 +1,55 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html
+  lang="en"
+  class="card"
+  data-qa="card">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=200px, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div
+      class="card__content"
+      data-qa="card"
+    >
+      <img
+        class="card__image"
+        src="images/imac.jpeg"
+        alt="imac"
+      >
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+      <p class="card__code">Product code: 195434</p>
+      <div class="card__review">
```

Comment:
```suggestion
      >
      
      <h2 class="card__title">
        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
      </h2>
      
      <p class="card__code">Product code: 195434</p>
      
      <div class="card__review">
```
fix code style all over the page